src/test/java/Domain/Game/TeamStub.java

```diff
@@ -1,33 +1,41 @@
 package Domain.Game;
 
-import Domain.Users.TeamOwner;
+import Domain.Users.*;
 
 import java.util.ArrayList;
 import java.util.List;
 
+
 public class TeamStub extends Team{
 
-    private List<TeamOwner> teamOwners;
```

Comment:
Why not keeping that to make tests easier? In my branch I added lists of coaches, players, etc in order to add items to the team without needing to pass the team owner as a parameter.

Reply:
will be inserted as part of Nir's code